docs/source/user_guide/pipeline-editor-configuration.md

```diff
@@ -0,0 +1,91 @@
+<!--
+{% comment %}
+Copyright 2018-2023 Elyra Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+## Configuring the pipeline editor
+
+### Configuring supported runtimes
+
+The pipeline editor supports three runtimes: Kubeflow Pipelines, Apache Airflow, and local execution in JupyterLab. By default, support for all runtimes is enabled when you [install Elyra](../getting_started/installation.md). The JupyterLab launcher window under the _Elyra_ category includes a tile for each enabled runtime:
+
+![Pipeline editor tiles in the JupyterLab launcher](../images/user_guide/pipeline-editor-configuration/pipeline-editor-tiles-in-jupyterlab-launcher.png)
+
+If you are planning to use only one runtime to execute pipelines, you can disable the other runtimes and hide their launcher tiles.
+
+#### Disabling runtimes
+
+Runtimes can be disabled by overriding the Elyra default configuration. 
+
+##### Override default using command line parameters
+
+To disable one or more runtimes, launch JupyterLab with the Elyra-specific `--PipelineProcessorRegistry.runtimes` parameter:
```

Comment:
I find this wording a bit confusing.  It speaks about _disabling_ runtimes, but then references the CLI option to _enable_ a runtime.

I wonder if rephrasing this to something like _Enabling specific runtimes_ and stating within the section contents 
 that this essentially _disables_ the other runtimes might be less confusing.

Reply:
Done. It should be now easier to read.

Reply:
Looks good - thanks!